From 6214113c8a2976e1fc421f3d825325a7f253d7e9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Thu, 20 Jun 2019 22:31:49 +0200
Subject: [PATCH 13/51] Fix warning, remove unsigned int < 0 check

Summary:
The bindings return an unsigned int value (with 101% denoting
an undeterminate value), checking for < 0 is pointless.

Reviewers: lukas, jgrulich, ngraham

Reviewed By: jgrulich

Differential Revision: https://phabricator.kde.org/D21974
---
 src/declarative/pkupdates.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/declarative/pkupdates.cpp b/src/declarative/pkupdates.cpp
index 34bcf4f..5a5e6ce 100644
--- a/src/declarative/pkupdates.cpp
+++ b/src/declarative/pkupdates.cpp
@@ -352,7 +352,7 @@ void PkUpdates::onPackageUpdating(PackageKit::Transaction::Info info, const QStr
 
     const uint percent = m_installTrans->percentage();
 
-    if (percent >= 0 && percent <= 100) {
+    if (percent <= 100) {
         setStatusMessage(i18nc("1 installation status, 2 pkg name, 3 percentage", "%1 %2 (%3%)",
                                PkStrings::infoPresent(info), PackageKit::Daemon::packageName(packageID), percent));
     } else {
-- 
2.28.0

